docs/source/getting_started/tutorials/ecms_tutorial.rst

```diff
@@ -0,0 +1,1054 @@
+        legend=False,
+        remove_background=True, #should remove the minimum value from the cut dataset, but it is possible somewhere to actually 
+        # the background (possibly where you define the dataset)
+    )
+    meas_b[3].plot_vs_potential(
+        mol_list=["H2", "O2"],
+        unit="pmol/s/cm^2",
+        logplot=False,
+        legend=False,
+        remove_background=True,
+        axes=axes_b, #to reuse the axis for co-plotting two separate parts of the measurement
+        linestyle="--",
+    )
+    axes_b[0].set_ylabel("cal. sig. / [pmol s$^{-1}$cm$^{-2}$]")
+    axes_b[0].set_xlabel(meas_b.U_name)
+    
+    fig_b = axes_b[0].get_figure()
+    fig_b.savefig("fig_b.png")
+
+
+
+.. image:: EC-MS_ixdat_tutorial_files/EC-MS_ixdat_tutorial_59_0.png
+
+
+EC-MS plot with carrier gases on right y-axis (uncalibrated) and products on left (calibrated)
+----------------------------------------------------------------------------------------------
+
+Note that it is not possible to mix linear and logarithmic scales in one
+plot with ixdat.
+
+.. code:: ipython3
+
+    meas_c = good_bit.cut(tspan=[5035, 5200])
+    meas_c.tstamp += meas_c.t[0]
+    
+    meas_c.set_bg(
+       tspan_bg=[0, 20], mass_list=["M2", "M32"]
+    )
+    
+    axes_c = meas_c.plot_measurement(
+        mol_lists = [[], ["H2", "O2"]], # left y-axis
+        logplot=False,
+        legend=False,
+    )
+    meas_c.plot_measurement(
+       mass_list=["M4", "M28", "M40"],  # right y-axis
+        #unit=["nmol/s"],  # [left, right] y-axes
+        logplot=False,
+        legend=False,
+        axes=axes_c, #to reuse the axis for co-plotting two separate parts of the measurement
+        #linestyle="--",
+    )
+    axes_c[2].set_ylabel("cal. sig. / [nmol s$^{-1}$]")
+    # axes_c[-1].set_ylim(bottom=-5)
+    
+    fig_c = axes_c[0].get_figure()
+    fig_c.savefig("fig_c.png")
+
+
+
+.. image:: EC-MS_ixdat_tutorial_files/EC-MS_ixdat_tutorial_61_0.png
+
+
+EC-MS plot with a third with system parameters added
+----------------------------------------------------
+
+In some cases it can be interesting to co-plot some other data,
+e.g. system parameters like iongauge pressure or MFC flow rates. Let’s
+have a look at the iongauge pressure during one of the CV cycles and
+plot that in a third panel of our regular EC-MS plots.
+
+.. code:: ipython3
+
+    from matplotlib import pyplot as plt
+    from matplotlib import gridspec
+    
+    # since we want to co-plot system parameters here, which were not part of the exported dataset, we need to use another version
+    # of our dataset # imported above, which hasn't been calibrated yet, so let's quickly calibrate that now
+    meas_part.add_calibration(loaded_cal)
+    
+    
+    # cut the relevant part from a big dataset
+    meas_d = meas_part.cut(tspan=[5035, 5200])
+    
+    # grab the iongauge pressure
+    ig1 = meas_d.grab("Iongauge value [mbar]")
+    
+    # create a figure with 3 panels
```

Comment:
Soon, you could use `ixdat.plotters.base_mpl_plotter.MPLPlotter.new_three_panel_axes()` for this :) 
https://github.com/ixdat/ixdat/blob/785bf2e0cab0ec8069fb8bcbf25104fc3d02dfca/src/ixdat/plotters/base_mpl_plotter.py#L91